packages/apt/test.sh

```diff
@@ -185,6 +187,26 @@ if [ "${can_upgrade}" = "yes" ]; then
   run_test
 fi
 
+echo "::group::Downgrade"
+
+if [ "${can_downgrade}" = "yes" ]; then
+  pgroonga_current_version=$(apt-cache search --full \^${pgroonga_package}\$ | \
```

Comment:
Can we use `apt info XXX` instead of `apt-cache search --full`?

Reply:
I used `apt info` instead of `apt-cache search --full`